addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -34,11 +35,12 @@ def default_get(self, fields_list):
     res_model_id = fields.Many2one(
         'ir.model', string="Applies to",
         compute="_compute_res_model_id", compute_sudo=True,
-        ondelete="cascade", precompute=True, readonly=False, required=True, store=True)
-    res_model = fields.Char("Model", readonly=True, required=True)
+        ondelete="cascade", precompute=True, readonly=False, required=False, store=True)
+    res_model = fields.Char("Model", readonly=False, required=False)
```

Comment:
Shouldn't res_model be required when we have a plan ? Not sure we want to have "model free" plans, do we ?